### PR TITLE
mtail: 3.0.0-rc43 -> 3.0.0-rc46

### PR DESCRIPTION
**Pull request by bbigras**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123544` at 379582b3 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>mtail</li>
    <li>tests.trivial</li>
  </ul>
</details>

File: pkgs/servers/monitoring/mtail/default.nix [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
   pname = "mtail";
-  version = "3.0.0-rc43";
+  version = "3.0.0-rc46";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "mtail";
     rev = "v${version}";
-    sha256 = "1lw7gz6fh3z7j4hin4q6y3wxxl4sgd4li1c0z60hmvfs671n6dg9";
+    sha256 = "sha256-/PiwrXr/oG/euWDOqcXvKKvyvQbp11Ks8LjmmJjDtdU=";
```
Comment (glittershark) [inline pkgs/servers/monitoring/mtail/default.nix:11]: Was this hash generated by unstable nix? Can we make this a stable nix hash instead?

**Review by glittershark — CHANGES_REQUESTED** [review]

File: pkgs/servers/monitoring/mtail/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/servers/monitoring/mtail/default.nix:11]: Stable nix also understands them so it doesn't matter.

Comment (bbigras) [reply pkgs/servers/monitoring/mtail/default.nix:11]: @glittershark Any reason you prefer stable nix hashes if the unstable ones work?

Note that I get unstable nix hashes when I use `nix-update` (which make updating a package painless for me) on my computer.

**Pull request merged**